README.md

```diff
@@ -53,7 +67,14 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+var weather;
+undefined. This expression is undefined because you have only declared a variable, but not defined its value yet.
+
+weather = "sunny";
+"sunny". Here we are assigning the value of "sunny" to the variable weather. Not sure why the console returns "sunny" though.
```

Comment:
That's just how `=` works - it evaluates to whatever value it's assigning.Try it out sometime!